board/piksiv3/rootfs-overlay/etc/init.d/S80zmq_router_sbp

```diff
@@ -5,5 +5,13 @@ cmd="zmq_router -f /etc/zmq_router/sbp_router.yml"
 dir="/"
 user=""
 
-source /etc/init.d/template_process.inc.sh
+touch /persistent/config.ini
+
+grep_results=$(cat /persistent/config.ini | grep "smoothpose_enable=True")
 
+if [ $? -eq 0 ]
+then
+  cmd="zmq_router -f /etc/zmq_router/sbp_router_smoothpose.yml"
```

Comment:
Suggest:

```bash

if grep -q "smoothpose_enable=True" /persistent/config.ini; then
  router_config="/etc/zmq_router/sbp_router_smoothpose.yml"
else
  router_config="/etc/zmq_router/sbp_router.yml"
fi

cmd="zmq_router -f $router_config"
```

Reply:
this is way more elegant. taking the change exactly and retesting.